python/vowpalwabbit/pyvw.py

```diff
@@ -137,7 +137,7 @@ def get_prediction(ec, prediction_type):
     ----------
 
     ec : Example
-    prediction_type : integer
+    prediction_type : enumerate
```

Comment:
I meant to use the enum in the example

Reply:
I didn't understand how can we provide enumerate as an argument as it will produce an object and also needs an iterable argument to be passed in enumerate.

Reply:
My original comment was on:
```python
>>> pyvw.get_prediction(ex, pylibvw.vw.pSCALAR)
vs 
>>> pyvw.get_prediction(ex, 0)
```

I believe it should be:
```python
>>> pyvw.get_prediction(ex, pylibvw.vw.pSCALAR)
```

Reply:
Thanks! I misunderstood it.